### PR TITLE
Fix Functions unit tests for Catalyst

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

File: Functions/Example/Tests/FUNSerializerTests.m [hunk 1]
```diff
@@ -82,7 +82,12 @@ - (void)testDecodeInvalidLong {
   FUNSerializer *serializer = [[FUNSerializer alloc] init];
   NSDictionary *input = @{
     @"@type" : @"type.googleapis.com/google.protobuf.Int64Value",
+#if TARGET_OS_MACCATALYST
+    // NSNumberFormatter is more robust? on Catalyst
+    @"value" : @"some junk",
```
Comment (bklimt) [inline Functions/Example/Tests/FUNSerializerTests.m]: I don't think this is the right way to fix this. The original value of this was invalid and should fail, even if that means not using NSNumberFormatter as-is. Let me think on it some more.

Comment (paulb777) [reply Functions/Example/Tests/FUNSerializerTests.m]: Maybe a bug in the beta related to  https://developer.apple.com/documentation/foundation/nsnumberformatter/1415137-defaultformatterbehavior?language=objc? I tried setting to https://developer.apple.com/documentation/foundation/nsnumberformatterbehavior/nsnumberformatterbehavior10_4?language=objc, but no difference.

Comment (ryanwilson) [reply Functions/Example/Tests/FUNSerializerTests.m]: Just wanted to check in and see if this still happens on the public version of Catalina and what the right approach moving forward should be (if `NSNumberFormatter` is in fact more robust through Catalyst.

Comment (ncooke3) [reply Functions/Example/Tests/FUNSerializerTests.m:87]: So running this test on catalyst in Xcode 12 (and Xcode 13) with no modifications passes. Im thinking this means that this was likely a bug in the beta this issues surfaced in and is no longer an issue– WDYT? @ryanwilson

Comment (ryanwilson) [reply Functions/Example/Tests/FUNSerializerTests.m:87]: Do you mean you ran the tests as is (without this PR) and everything was fine?

Comment (ncooke3) [reply Functions/Example/Tests/FUNSerializerTests.m:87]: Yep, exactly.

Comment (ryanwilson) [reply Functions/Example/Tests/FUNSerializerTests.m:87]: Wicked! :shipit: thanks for checking

**Comment by ryanwilson** [comment]
As @ncooke3 mentioned, this must have been a beta issue as the failure doesn't reproduce in shipping versions of Xcode 12 or 13.

**Pull request closed without merging**